ec2-cluster/main.tf

```diff
@@ -0,0 +1,48 @@
+resource "random_id" "instance_id" {
+  byte_length = 4
+}
+
+locals {
+  cluster_id = random_id.instance_id.hex
+}
+
+module "nodes" {
+  source               = "./modules/node"
+  instance_count       = var.instance_count
+  cluster_id           = local.cluster_id
+  env                  = var.env
+  workspace            = terraform.workspace
+  instance_type        = var.instance_type
+  disk_size            = var.node_disk_size
+  tessellation_version = var.tessellation_version
+  instance_keys        = var.instance_keys
+  public_port          = var.public_port
+  p2p_port             = var.p2p_port
+  cli_port             = var.cli_port
```

Comment:
Please prefix these with `l0_` to be consistent with what's underneath for `l1_`.